dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java

```diff
@@ -153,16 +154,14 @@ private static String extractHTTPPrefix(String gitUrl, String reference, String
         // tool specific
         if (container instanceof io.dockstore.webservice.core.Tool) {
             io.dockstore.webservice.core.Tool inputTool = (io.dockstore.webservice.core.Tool)container;
-            tool.setToolname(inputTool.getToolname());
+            tool.setToolname(Strings.isNullOrEmpty(inputTool.getToolname()) ? Strings.nullToEmpty(inputTool.getName()) : Strings.nullToEmpty(inputTool.getToolname()));
```

Comment:
Let's change this, should be `dockstore-biobambam2/bamtofastq` in the case of  `quay.io/wtsicgp/dockstore-biobambam2/bamtofastq`